dts/bindings/clock/ti,mspm0-ckm-bus.yaml

```diff
@@ -0,0 +1,39 @@
+# Copyright (c) 2025 Texas Instruments Inc.
+# Copyright (c) 2025 Linumiz
+# SPDX-License-Identifier: Apache-2.0
+
+description: TI MSPM0 Clock Module Bus
```

Comment:
There is no such thing as a clock module bus anywhere in the datasheet. This is also wrong semantically, as it would mean there is a "bus" transporting clock(s). 

This need to be renamed. Actually, I don't see much issue with something more generic as "TI MSPM0 Clock"